### ISSUE TITLE
Overlapping identities between LFW and VGGface2

### ISSUE THREAD

**Issue by maria8899** [deepinsight/insightface#101]
Hi,
In your paper you report results on LFW when trained on VGGface2 in tables 2, 3, 4, 5 and 8, but I think there are quite a lot of overlapping identities between LFW and VGGface2 as both are based on celebrities. Have you checked for overlapping identities? Have you removed them when training? If not, it does not seem quite fair to report those numbers, or maybe it should be mentioned in the paper?

Also, when you say you are training with VGGface2, are you using the train and test set of VGGface2 (i.e. 8,631 +  500  identities) or just the train set of VGGface2 (8,631)?

**Comment by nttstar** [comment]
Training set only. VGGFace2 are just used for comparing network backbones so only relative scores are considered.
